docs/docker/README.md

```diff
@@ -31,8 +33,18 @@ $ docker pull mcr.microsoft.com/playwright:bionic
 
 ### Run the image
 
+By default, the Docker image will use the `root` user to run the browsers. This will disable the Chromium sandbox which is not available with root. If you run trusted code (e.g. End-to-end tests), then the root user is in most cases fine. For web scraping or crawling, we recommend to create a separate user inside the Docker container and use the seccomp profile.
+
+#### Trusted environments
+
 ```
-$ docker container run -it --rm --ipc=host --security-opt seccomp=seccomp_profile.json mcr.microsoft.com/playwright:bionic /bin/bash
+docker run -it --rm mcr.microsoft.com/playwright:bionic /bin/bash
+```
+
+#### With the Chromium Sandbox
+
+```
+$ docker run -it --rm --ipc=host --security-opt seccomp=seccomp_profile.json mcr.microsoft.com/playwright:bionic /bin/bash
```

Comment:
I think ipc=host is also applicable to trusted environments.

Reply:
good catch!